pyface/application.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) 2014-2016 by Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+
+""" Define a base application class to create the event loop, and launch
+the creation of application windows.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import logging
+import sys
+from traceback import format_exception, format_exception_only
+
+from traits.api import Instance, List, Tuple, Vetoable
+
+from .base_application import ApplicationEvent, BaseApplication
+from .i_gui import IGUI
+from .i_image_resource import IImageResource
+from .i_splash_screen import ISplashScreen
+from .i_window import IWindow
+
+logger = logging.getLogger(__name__)
+
+
+def clear_handlers(logger):
+    """ Remove all handlers from the given logger. """
+    for handler in logger.handlers[::-1]:
+        if hasattr(handler, 'close'):
+            handler.close()
+        logger.removeHandler(handler)
+
+
+class Application(BaseApplication):
+    """ A basic Pyface application.
+
+    This handles setting up starting up the GUI, window management, logging,
+    and other common features that we want when creating a GUI application.
+
+    Often you will want to subclass this application, but it can be used as-is
+    by hooking a listener to the `application_initialized` event::
+
+
```

Comment:
I would add the imports here `from pyface.api import ApplicationWindow, HeadingText`